java/private/spotbugs_config.bzl

```diff
@@ -1,17 +1,70 @@
+
+def spotbugs_binary(
```

Comment:
Could we expose this through `//java:defs.bzl`?

Reply:
Yes!